src/pages/Profile.js

```diff
@@ -78,7 +78,10 @@ function Profile({
 
   useEffect(() => {
     if (!profiles.isLoading) {
-      dispatch({ type: 'selectedCountry', selectedCountry: profiles.profile });
+      dispatch({
+        type: 'selectedCountry',
+        selectedCountry: profiles.parent ? profiles.parent : profiles.profile
```

Comment:
If you're at the country level, `profiles.parent` would point to `Africa` me thinks... we may need to use `geoId` and `countries` list to pick the `selectedCountry`

Reply:
I'll check level but i know we don't like to support `https://dev.dominion.africa/profile/country-ZA` but it's possible to visit it.